src/components/notification/NotificationHelpers.ts

```diff
@@ -0,0 +1,39 @@
+import { AssessmentCategories } from '../assessment/assessmentShape';
+import { Notification } from './notificationShape';
+
+type filterByIdOptions = {
+  assessment_id?: number;
+  submission_id?: number;
+};
+
+type filterByTypeOptions = AssessmentCategories | 'Grading';
+
+export function filterNotificationsById(
+  notifications: Notification[],
+  opts: filterByIdOptions = {}
+) {
+  return notifications.filter(
+    n =>
+      (opts.assessment_id === undefined || n.assessment_id === opts.assessment_id) &&
+      (opts.submission_id === undefined || n.submission_id === opts.submission_id)
+  );
+}
+
+/*
+  For filtering notifications on the academy navigation bar.
```

Comment:
Remove this line. Its a helper function so we dont really need to say where it is used in the function itself.